src/workflow/ArcanistLandWorkflow.php

```diff
@@ -1028,7 +1028,23 @@ private function push() {
         'Holding change in **%s**: it has NOT been pushed yet.',
         $this->onto)."\n");
     } else {
-      echo pht('Pushing change...'), "\n\n";
+      $sirmixalot_enrolled = $this->getConfigFromAnySource(
+        'sirmixalot.enrolled',
+        false);
+      // check if repo is enrolled to sirmixalot
+      if ($sirmixalot_enrolled) {
+        // if repo is enrolled, land change on a specific remote branch
+        $sirmixalot_remote_landed_branch_templ = $this->getConfigFromAnySource(
+          'sirmixalot.remote_landed_branch_template',
+          'landed/%s');
```

Comment:
Default value will not be used if you will not register config to `src/configuration/ArcanistSettings.php`. Check my comment above.

If this config does not exist `$sirmixalog_remote_landed_branch_templ` will be null, and land workflow will fail.